src/main/scala/higherkindness/rules_scala/workers/common/FileUtil.scala

```diff
@@ -59,6 +59,29 @@ object FileUtil {
     path.getFileName().toString().stripPrefix("header_").stripPrefix("processed_")
   }
 
+  /**
+   * Given a Bazel path to the bazel output directory, return a path excluding the bazel configuration specific parts of
+   * the path. This is analogous to Bazel's File.short_path function. Fair warning: this function is super good enough,
+   * but is likely not perfect.
+   */
+  def bazelShortPath(path: Path, replaceExternal: Boolean = true): Path = {
+    val nameCount = path.getNameCount()
+    val pathString = path.toAbsolutePath().normalize().toString()
+
+    val shortPath = if (path.startsWith("bazel-out") && nameCount >= 4) {
+      path.subpath(3, nameCount)
+    } else {
+      path
+    }
+
+    // Handle difference between Bazel's external directory being referred to as .. in the short_path
+    if (replaceExternal && shortPath.startsWith("external")) {
```

Comment:
A more correct way of doing this might be:
```scala
val shortPathNameCount = shortPath.getNameCount()

if (shortPathNameCount > 0 && shortPath.subpath(0, 1) == Paths.get("external")) {
  Paths.get("..", shortPath.subpath(1, shortPathNameCount))
} else {
  shortPath
}
```

Reply:
Upon first reading I like this.

I think `shortPathNameCount > 0 && shortPath.subpath(0, 1) == Paths.get("external")` is equivalent to `shortPath.startsWith("external")`, so I don't think I'll change that.

I'll play around with the Paths.get approach to avoid the string operations.

Reply:
Agreed, yes. I realized that when working on this yesterday and ended up using your suggestion.